### ISSUE TITLE
The performance is poor. I train my dataset on 8 GPUs for 200epoches 

### ISSUE THREAD

**Issue by XUYUNYUN666** [jfzhang95/pytorch-deeplab-xception#136]
![image](https://user-images.githubusercontent.com/38391161/69422761-7b4e3a80-0d5f-11ea-8cac-e0af53c1424b.png)
my dataset is put the folder like cityscapes, the label is grayscale with single channel, the parameters is below:
datset:cityscapes
backbone:resnet
out_stride:16
lr:0.01
lr_scheduler:poly
loss_type:ce
epoch:200
base_size:1024
crop_size:513

**Comment by FantasyJXF** [comment]
@XUYUNYUN666  Trained with COCO2017, got same result with you. Have you figure out the question why the learning curve like this?

**Comment by jfzhang95** [comment]
I have update the code, please try again.

**Comment by tjucwb** [comment]
So far I also encounter this problem, how could I solve it, I used the default setting in the code with freezing bn
@XUYUNYUN666 @jfzhang95 
Thank u guys very much

**Comment by bberry25** [comment]
Hi, can I get ur code for visualization of training result?
